src/app/mapView/MapView.jsx

```diff
@@ -142,7 +142,7 @@ export default class MapView extends Component {
             {
               this.state.suggestions && <Dropdown options={
                 this.state.suggestions.map(organization => ({
-                  key: organization.id,
+                  id: organization.id,
```

Comment:
I fixed this because the Dropdown was looking for an id property, and was complaining because no id property was specified. You can see this in its proptypes:

```
Dropdown.propTypes = {
  options: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.string.isRequired,
    label: PropTypes.string.isRequired,
    onClick: PropTypes.func.isRequired,
  })).isRequired,
```